### DISCUSSION TITLE
Cannot apply, camera UI is not working at all

### DISCUSSION THREAD

**Discussion by an1dee3301** [community/community#118512] [GitHub Education]
### Hi everyone,

I can't take photos with the form, the webcam is not showing. When I try on my phone it just stuck on the front camera and I can't do anything to change it to the back camera, can somebody help?

**Comment by choidongjune** [comment]
you can check the button which can upload the image

**Comment by samus-aran** [comment]
Hi @andee3301 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by MadeByUche** [comment]
MSFT should be absolutely embarrassed to have this thing be so flawed.
